src/openrct2/object/RideObject.cpp

```diff
@@ -763,6 +767,24 @@ rct_ride_entry_vehicle RideObject::ReadJsonCar(const json_t* jCar)
     car.draw_order = ObjectJsonHelpers::GetInteger(jCar, "drawOrder");
     car.num_vertical_frames_override = ObjectJsonHelpers::GetInteger(jCar, "numVerticalFramesOverride");
 
+    auto jAnimationSpeed = json_object_get(jCar, "animationSpeed");
+    auto animationMultiplier = ObjectJsonHelpers::GetInteger(
+        jAnimationSpeed, "multiplier", VehicleDefaultAnimationSpeeds[car.animation][1]);
+    auto animationExponent = ObjectJsonHelpers::GetInteger(
+        jAnimationSpeed, "exponent", ANIMATION_SPEED_MAX_EXPONENT - (VehicleDefaultAnimationSpeeds[car.animation][0]));
+
+    car.animation_speed_multiplier = animationMultiplier;
+    car.animation_speed_exponent = ANIMATION_SPEED_MAX_EXPONENT - animationExponent;
+
+    auto jSteamTranslation = json_object_get(jCar, "steamPositionModifier");
+    auto longitudeSteam = ObjectJsonHelpers::GetFloat(jSteamTranslation, "longitudinal", 1.0);
```

Comment:
Not exactly a fan of having floats around, different compilers and platforms may bring different results, there are no guarantees that this will perform exactly the same way. I recall removing the misc sprites from the network checksum but I rather have the simulation deterministic for all aspects.